Library/ViewModels/ProjectNavBarViewModel.swift

```diff
@@ -129,18 +144,27 @@ ProjectNavBarViewModelInputs, ProjectNavBarViewModelOutputs {
 
     self.categoryButtonTitleColor = self.categoryButtonTintColor
 
-    self.goToLoginTout = loggedOutUserTappedStar
+    self.goToLoginTout = loggedOutUserTappedSaveButton
 
-    self.showProjectStarredPrompt = projectOnStarToggleSuccess
+    self.showProjectSavedPrompt = project
+      .takeWhen(self.saveButtonTappedProperty.signal)
       .filter { $0.personalization.isStarred == true && !$0.endsIn48Hours(
-        today: AppEnvironment.current.dateType.init().date) }
-      .map { _ in Strings.project_star_confirmation() }
+        today: AppEnvironment.current.dateType.init().date ) }
+      .filter { _ in
+        !AppEnvironment.current.ubiquitousStore.hasSeenSaveProjectAlert ||
+        !AppEnvironment.current.userDefaults.hasSeenSaveProjectAlert
+      }
+      .on(value: { _ in
+        AppEnvironment.current.ubiquitousStore.hasSeenSaveProjectAlert = true
+        AppEnvironment.current.userDefaults.hasSeenSaveProjectAlert = true
+      })
+      .ignoreValues()
 
-    self.starButtonSelected = project
+    self.saveButtonSelected = project
       .map { $0.personalization.isStarred == true }
       .skipRepeats()
 
-    self.starButtonAccessibilityHint = self.starButtonSelected
+    self.starButtonAccessibilityHint = self.saveButtonSelected
```

Comment:
I think the hint is redundant since the label is "Save project". I think we can remove the hint here.